sonar/modules/documents/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -115,7 +115,8 @@
               }
             },
             "startDate": {
-              "type": "date"
+              "type": "date",
+              "format": "yyyy-MM-dd||yyyy"
```

Comment:
are you sure about `||` ? Isn't it `|` in regular expression ?

Reply:
Yes I'm sure --> https://www.elastic.co/guide/en/elasticsearch/reference/current/date.html#multiple-date-formats